packages/assistify-ai/server/lib/SmartiAdapter.js

```diff
@@ -88,7 +88,12 @@ export class SmartiAdapter {
 		if (conversationId) {
 			SystemLogger.debug(`Conversation ${ conversationId } found for channel ${ message.rid }`);
 			// add message to conversation
-			SmartiProxy.propagateToSmarti(verbs.post, `conversation/${ conversationId }/message`, requestBodyMessage);
+			const res = SmartiProxy.propagateToSmarti(verbs.post, `conversation/${ conversationId }/message`, requestBodyMessage);
+			// mark message as synced
+			if (res) {
+				SystemLogger.debug('Conversation found and message will be synced now');
+				Meteor.call('markMessageAsSynced', message._id);
```

Comment:
This could be executed asynchronously. Meteor methods called on the server are executed asynchronously. Or might this interfer with a selection of documents to be synced?
```
Meteor.defer(()=>Meteor.call('markMessageAsSynced', message._id));
```